W10ANDW11-NETWORK-TCP-BACKSUBOPTIMIZATION.ps1

```diff
@@ -1,65 +1,65 @@
-<# 
+<#
 .SYNOPSIS
-    This script change all settings to default, that are optimized by the "W10ANDW11-NETWORK-TCP-DESUBOPTIMIZATION" skript.   
- 
- .NOTES 
+    This script change all settings to default, that are optimized by the "W10ANDW11-NETWORK-TCP-DESUBOPTIMIZATION" skript.
+
+ .NOTES
     Version:        0.09
     Author:         MysticFoxDE (Alexander Fuchs)
     Creation Date:  22.02.2023
 
-.LINK 
+.LINK
     https://administrator.de/tutorial/wie-man-das-windows-10-und-11-tcp-handling-wieder-desuboptimieren-kann-5529700198.html#comment-5584260697
     https://community.spiceworks.com/topic/post/10299845
     https://www.golem.de/news/tcp-die-versteckte-netzwerkbremse-in-windows-10-und-11-2302-172043.html
 #>
 
 # PROMPT THE USER TO ELEVATE THE SCRIPT
-# Great thanks to "Karl Wester-Ebbinghaus/Karl-WE" for this very useful aid.   
-if (-not (New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)) 
+# Great thanks to "Karl Wester-Ebbinghaus/Karl-WE" for this very useful aid.
+if (-not (New-Object Security.Principal.WindowsPrincipal ([Security.Principal.WindowsIdentity]::GetCurrent())).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator))
 {
   $arguments = "-NoExit -ExecutionPolicy Bypass -File `"$($myInvocation.MyCommand.Definition)`""
   Start-Process powershell -Verb runAs -ArgumentList $arguments
   exit
 }
 
 #SET ALL NIC ADVANCED SETTINGS TO DEFAULT
-Get-Netadapter -Physical | Reset-NetAdapterAdvancedProperty -DisplayName "*"
+Get-NetAdapter -Physical | Reset-NetAdapterAdvancedProperty -DisplayName "*"
 
 # SET RSS TO DEFAULT
-netsh int tcp set global RSS=Enabled
+netsh int tcp Set-Variable global RSS=Enabled
 Set-NetOffloadGlobalSetting -ReceiveSideScaling Enabled
 
 # SET RSC TO DEFAULT
-netsh int tcp set global RSC=Enabled
+netsh int tcp Set-Variable global RSC=Enabled
 Set-NetOffloadGlobalSetting -ReceiveSegmentCoalescing Enabled
 
 # SET PACKET COALESCING TO DEFAULT
 Set-NetOffloadGlobalSetting -PacketCoalescingFilter Enabled
 
 # SET TCP CONGESTION CONTROL TO DEFAULT
-netsh int tcp set supplemental template=Datacenter congestionprovider=CUBIC
-netsh int tcp set supplemental template=Datacentercustom congestionprovider=CUBIC
-netsh int tcp set global ECN=Disabled
+netsh int tcp Set-Variable supplemental template=Datacenter congestionprovider=CUBIC
```

Comment:
netsh and set-variable? i think the powershell beautifier sets this?

Reply:
good spot. yeah, the beautifier did this :( 
I reverted that commit

Reply:
"netsh int tcp Set-Variable" ist wrong